src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -172,6 +175,8 @@ uct_rc_verbs_iface_poll_tx(uct_rc_verbs_iface_t *iface)
         ucs_arbiter_group_schedule(&iface->super.tx.arbiter,
                                    &ep->super.arb_group);
         uct_rc_verbs_update_tx_res(&iface->super, ep, count);
+        ucs_arbiter_dispatch(&iface->super.tx.arbiter, 1, uct_rc_ep_process_pending,
+                             NULL);
```

Comment:
why needed? It is called one line above

Reply:
we removed it from `uct_rc_verbs_update_tx_res`